.github/workflows/ruby_tests.yml

```diff
@@ -0,0 +1,127 @@
+name: Ruby tests
+on:
+  push:
+    branches: [master, foreman_2_1, foreman_1_24]
+  pull_request:
+    branches: [master, foreman_2_1, foreman_1_24]
+
+defaults:
+  run:
+    working-directory: ./projects/foreman
+
+jobs:
+  test_ruby:
+    env:
+      PGHOST: 127.0.0.1
+      PGUSER: foreman
+      RAILS_ENV: test
+
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: quay.io/jomitsch/postgres-with-evr
+        ports: ['5432:5432']
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        env:
+          POSTGRES_USER: foreman
+          POSTGRES_PASSWORD: foreman
+
+    steps:
+      - name: Checkout foreman_rh_cloud repo
+        uses: actions/checkout@v2
+        with:
+          path: ${{ github.workspace }}/projects/foreman_rh_cloud
+      - name: Checkout Foreman repo
+        uses: actions/checkout@v2
+        with:
+          repository: theforeman/foreman
+          path: ${{ github.workspace }}/projects/foreman # checkout Foreman and foreman_rh_cloud as siblings, same as dev env
+          ref: 1.24-stable
+      - name: Checkout Katello repo
+        uses: actions/checkout@v2
+        with:
+          repository: Katello/katello
+          path: ${{ github.workspace }}/projects/katello # checkout Foreman and foreman_rh_cloud as siblings, same as dev env
+          ref: KATELLO-3.14
+      - name: check for postgres
+        run: file /usr/share/postgresql/12/extension/
+      - name: Cache bundler
+        id: cache-bundler
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-1_24-${{ hashFiles('**/Gemfile.lock') }}
```

Comment:
```suggestion
          key: ${{ runner.os }}-gems-1_24-${{ hashFiles('projects/foreman_rh_cloud/config/Gemfile.lock.gh_test') }}
```

Reply:
it will take it from the default foreman folder, see line 10: https://github.com/theforeman/foreman_rh_cloud/pull/436/files#diff-ea6a337c44ed785f53ad619e854ad6b32b18f146361d556035cee38bc4e005a2R10

Reply:
But there is no Gemfile.lock in any repository. We only create the lock file a couple of steps later. Here we can only use `Gemfile.lock.gh_test` file which resides in our folder. You can leave the `**/` glob specifier, but only one file exists that will match this glob.